Automattic/jetpack
# Memberships: remove remaining token subscription service code

Fixes 264-gh-automattic/gold

## Proposed changes:

* Removes a condition to use a class which no longer exists.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply D134609-code to your sandbox
* Make sure paywall-protected protected content still works with patch applied.
